HardwareDeployment.md

```diff
@@ -0,0 +1,116 @@
+# Hardware Deployment
+
+Andreas Antonopoulos tells us "Not your keys, not your Bitcoin".  Here's another Bitcoin axiom "Not your hardware, not your node". 
+
+In order for your Bitcoin experience to be truly self-sovereign and trustless you should consider running your node(s) on your own hardware and internet connection. BTCPayServer is an excellent way to run Bitcoin & Lightning nodes because not only are you validating the network you will also get the ability to accept base layer Bitcoin and Lightning payments.  
+To that end here are instructions to install and host your very own BTCPay Server. 
+
+The process is basically the following:
+
+1. Purchase and assemble hardware. 
+2. Install base Operating System and configure networking.
+3. Install BTCPayServer-Docker.
+
+While it may be possible to install BTCPayServer or parts of it's architecture on ARM based single board computers (ie. Raspberry Pi, 
+Odroid, OrangePi or NanoPC) the process may require additional skills and knowledge that are beyond the scope of this tutorial. 
+For the sake of speed, compatibility and ease of installation it is recommended that you use an Intel/AMD based computer with a minimum of 4Gb RAM and a 500 Gb hard drive.  The Gigabyte Brix or Intel NUC lines of compact computers are a good place to start. They are basically small, fanless boxes built using laptop components. 
+
+BTCPayServer can be successfully installed on the following hardware.  The end result is a small, quiet node that is fast enough to sync from block zero.  The total cost is approximatly $225 (including tax and delivery).  These are "off the shelf" components that can be purchased from any online retailer (TigerDirect, NewEgg, Amazon, BestBuy etc).   It will take roughly 24 hours to sync the Bitcoin blockchain. You can save little money by cutting the RAM and hard drive size in half. 
+ 
+1. Gigabyte Brix GB-BXBT-1900
+![alt text](https://i1.wp.com/lightninginabox.co/wp-content/uploads/2018/08/20150603180458_m.png?resize=300%2C254&ssl=1 "Gigabyte Brix GB-BXBT-1900")
```

Comment:
Can you add alt text for all 3 images, so that our docs page does not show "alt text" in image description?

1. ![GB-BXBT-1900] 
2. ![DDR3L-1600] 
3. ![1TB Mobile Hard Disk Drive]